Projects/AccountSDKIOSWebExample/AnotherSharedKeychainApp/AnotherSharedKeychainAppApp.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  AnotherSharedKeychainAppApp.swift
+//  AnotherSharedKeychainApp
+//
+//  Created by Daniel Echegaray on 2021-11-21.
+//
+
+import SwiftUI
```

Comment:
Currently, the Projects folder is included in our SPM package. Maybe it is a good moment to exclude it? Adding another app increases final size of package

Reply:
So its included in AccountSDKIOSWebExample, mhm